app/models/concerns/hyrax/solr_document/metadata.rb

```diff
@@ -52,6 +52,7 @@ def self.coerce(input)
         attribute :read_groups, Solr::Array, ::Ability.read_group_field
         attribute :collection_ids, Solr::Array, 'collection_ids_tesim'
         attribute :admin_set, Solr::Array, "admin_set_tesim"
```

Comment:
Should there be an attribute added for `admin_set_sim`?

Reply:
Not completely sure.  Title has `_tesim` and `_sim`.  It only has [`title_tesim` in this file](https://github.com/samvera/hyrax/blob/main/app/models/concerns/hyrax/solr_document/metadata.rb#L60).  There are other `_sim` properties indexed, but none of them appear in this file.

Reply:
I'm assuming that the set of attributes are those that can be in the solrdoc for search results.  Since only _tesim is returned in the solrdoc, the _sim version does not need to be an attribute.